CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+- Implement Server-Side Apply mode (https://github.com/pulumi/pulumi-kubernetes/pull/2029)
```

Comment:
Maybe add some more details on how to enable this. (And mark it preview etc.)

Reply:
I'll add that in a followup PR. This isn't really feature-complete yet, so I'm not quite ready to encourage usage.